MediaWikiTelegramBot.js

```diff
@@ -65,53 +80,71 @@ tgBot.onText( /^\/setlanguage ?(.*)/, function ( msg, match ) {
     debug( userID, "newLanguageCode is " + newLanguageCode );
     debug( userID, "newLanguageCode type is " + typeof( newLanguageCode ) );
 
+    // Check if a language code was provided
     if ( newLanguageCode === '' ) {
         tgBot.sendMessage( userID, "The current language code is " +
             getLanguageCode( userID )
         );
 
+        // TODO - what if the language code is invalid?
```

Comment:
Indeed :)

Should be checked with the MediaWiki API. I believe there's an API function that can verify it.